src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -256,7 +257,7 @@ public Distance getElevatorHeight() {
   public Command goUp(Supplier<Double> upPower) {
     return runEnd(
         () -> {
-          elevatorMotor1.set(Math.abs(upPower.get()) * elevatorOutput);
+          elevatorMotor1.set(upPower.get() * elevatorOutput);
```

Comment:
This should stay the absolute value, right? Otherwise it could go down when the function is goUp